bin/accessibility-automation/cypress/index.js

```diff
@@ -337,9 +329,11 @@ Cypress.Commands.add('performScan', () => {
         }
         cy.window().then(async (win) => {
             browserStackLog(`Performing accessibility scan`);
-            await performScan(win);
+            cy.wrap(performScan(win), {timeout:40000});
```

Comment:
This sets a fixed timeout, which can break the test if mochaTimeout is less than 40s

Reply:
if the response from performScan take less time no timeout will occur. Eventually if response does not come and mocha timeout occurs it should be considered as timeout.